### PR TITLE
CIV-12358 - Record click controller

### PR DESCRIPTION
**Pull request by m-meulendijks-v1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-12358

### Change description ###
Delete notification controller

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: dashboard-notifications/src/main/java/uk.gov.hmcts.reform/dashboard/entities/NotificationEntity.java [hunk 1]
```diff
@@ -10,33 +12,63 @@
...
+    @Size(max = 256)
+    @Column(name = "en_html", length = 256)
+    private String enHtml;
+
+    @Size(max = 256)
+    @Column(name = "cy_html", length = 256)
+    private String cyHtml;
+
+    @Size(max = 256)
+    @Column(name = "message_param", length = 256)
```
Comment (laarmada) [inline dashboard-notifications/src/main/java/uk.gov.hmcts.reform/dashboard/entities/NotificationEntity.java:56]: with that changes we need to update the migration files about create tables....
...created_by for example and every one.

**Review by laarmada — APPROVED** [review]

File: dashboard-notifications/src/main/java/uk.gov.hmcts.reform/dashboard/entities/NotificationEntity.java (hunk 1 above)
Comment (m-meulendijks-v1) [reply dashboard-notifications/src/main/java/uk.gov.hmcts.reform/dashboard/entities/NotificationEntity.java:56]: Do we? The changes are to the entities, but the database column names should still match due to the @Column.
With the old file, there seemed to be entity/database mismatches.

File: dashboard-notifications/src/resources/application.properties
Comment (kannan-v-hmcts) [inline dashboard-notifications/src/resources/application.properties:1]: Is this file needed ? Logging queries is already handled in the application.yaml file

File: dashboard-notifications/src/test/java/uk/gov/hmcts/reform/dashboard/services/DashboardNotificationServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    private DashboardNotificationService dashboardNotificationService;
+
+    @Mock
+    private NotificationRepository notificationRepository;
+
+    private final UUID id = UUID.randomUUID();
+
+    @BeforeEach
+    void setUp() {
```
Comment (kannan-v-hmcts) [inline dashboard-notifications/src/test/java/uk/gov/hmcts/reform/dashboard/services/DashboardNotificationServiceTest.java:26]: Can we use @InjectMocks rather than Initializing the service before every test ? Example can be found in TaskListServiceTest. I am also happy to leave it as is

File: dashboard-notifications/src/resources/application.properties
Comment (m-meulendijks-v1) [reply dashboard-notifications/src/resources/application.properties:1]: I've removed this file, I added it for testing before, we don't need it.

File: dashboard-notifications/src/test/java/uk/gov/hmcts/reform/dashboard/services/DashboardNotificationServiceTest.java (hunk 2 above)
Comment (m-meulendijks-v1) [reply dashboard-notifications/src/test/java/uk/gov/hmcts/reform/dashboard/services/DashboardNotificationServiceTest.java]: Done

**Review by kannan-v-hmcts — APPROVED** [review]
